pkg/fakerp/admin_handlers.go

```diff
@@ -124,3 +126,45 @@ func (s *Server) handleRotateSecrets(w http.ResponseWriter, req *http.Request) {
 	}
 	s.log.Info("rotated cluster secrets")
 }
+
+// handleGetLogLevel handles admin requests to get the configured OpenShift component log level for OpenShiftManagedClusters
+func (s *Server) handleGetLogLevel(w http.ResponseWriter, req *http.Request) {
```

Comment:
you don't need any fakerp changes or additional geneva action - with the config struct in place the generic admin API can handle getting and putting loglevel changes.

Reply:
alright